wagtail/contrib/frontend_cache/signal_handlers.py

```diff
@@ -1,7 +1,11 @@
 from django.apps import apps
 
 from wagtail.contrib.frontend_cache.utils import purge_page_from_cache
-from wagtail.core.signals import page_published, page_unpublished
+from wagtail.core.signals import page_moved, page_published, page_unpublished
+
+
+def page_moved_signal_handler(instance, **kwargs):
```

Comment:
Worth mentioning this separately in release notes.

Reply:
> I think we should also purge the URL of the page in it's previous location too.

That would be cool. How do you envisage this being supplied to signal handlers? would supplying `old_url_path` (defined currently in `Page.move()`) be enough here?

Reply:
@kaedroho I just realised something quite important. If a page has descendants, the paths/URLs of those descendants might also change as a result of `move()`. So, if we want to clear all outdated URLs from the cache, we really need the old URLs for all of those descendants, as well as that of the current page. Is it enough to provide a list of all outdated page URLs, or do you need the full page objects (in their 'before' state) in order to reliably clear things from the front-end cache?